file_paths.md

```diff
@@ -25,13 +25,23 @@ The files annotated by an '*' indicate that they are created by package installa
     /etc/puppetlabs/code *                # :codedir                     contents moved from :confdir
         environments *                    # :environmentpath
           production *
+            hieradata *
             environment.conf *
             manifests *
             modules *
         hiera.yaml *                      # :hiera_config
-        hieradata *                       # n/a
         modules *                         # user modulepath
 
+    /etc/puppetlabs/code-staging          # staging directory            n/a
```

Comment:
This would be the place where users are interacting with code. My understanding is that things that will have user interaction should usually go in /etc, and since we already have puppet code in /etc this seems like the right place for it to go?